bin/runtime-common/src/messages_xcm_extension.rs

```diff
@@ -163,3 +210,121 @@ impl<H: XcmBlobHauler> HaulBlob for XcmBlobHaulerAdapter<H> {
 			})
 	}
 }
+
+// TODO: use following structures in the `pallet-message-queue` configuration
+
+// TODO: it must be a part of `pallet-xcm-bridge-hub`
+pub struct LocalInboundXcmChannelSuspender<Origin, Inner>(PhantomData<(Origin, Inner)>);
```

Comment:
Open question: in https://github.com/paritytech/cumulus/pull/2157, the `QueueChangeHandler` of `pallet_message_queue` is not set (???) to the `cumulus-pallet-xcmp-queue`. Given that, our backpressure won't actually work - we are pausing dispatch here, but the transport channel will stay opened.

Reply:
This: https://github.com/paritytech/cumulus/pull/2157/files#diff-3d3febd8cfcd42ba5bb949d63fdafb9a71da3db1feb35a45601404b64fcd70c4R575 and e.g. this https://github.com/paritytech/cumulus/pull/2157/files#diff-447906ba563849a582953cc1753f58164c23d4902539e37c5dde387113a3aa5bR411